start.py

```diff
@@ -533,38 +536,48 @@ def _command_condition(deployment, command_id, command_description):
 
 # TODO: Get rid of the excess amount of code duplication between the following
 # function and _configure_kudu.
-def _configure_kafka(deployment, cluster, kafka_version):
+def _deactivate_install_kafka(deployment, cluster, kafka_version):
```

Comment:
Should we just have one method to deactivate Kafka and another to install Kafka? `deactivate_install_kafka` is a little confusing :)

Reply:
Done.